### ISSUE TITLE
Cursor Blink Rate

### ISSUE THREAD

**Issue by millera023** [jaredpar/VsVim#2445]
Is it possible to not tie the cursor blink rate to the operating system preference (As mentioned here: https://github.com/jaredpar/VsVim/wiki/faq#how-can-i-change-the-cursor-blink-rate)?

The reason I'm asking is because of the bug in windows 10 that is described here: https://www.reddit.com/r/Windows10/comments/4qlgqp/taskbar_icons_stay_highlighted_orange_forever/

**Comment by ricksladkey** [comment]
As is customary in vim, the cursor and the blink rate is configurable to a ridiculous degree! Emulating the whole `guicursor` option would be a fair amount of work, but conceptually, having a VsVim-specific setting that gets it default value from the operating system preference would be backward-compatible and yet allow configuration with a setting.

**Comment by lichray** [comment]
Should be given at least an option to disable blinking.  Setting it in control panel is too involved.

**Comment by in-just** [comment]
> Should be given at least an option to disable blinking. Setting it in control panel is too involved.

I disagree. Adding this feature would make it further convoluted at to the behavior of your cursor across the entire Operating System. I'm for a **one cursor definition** in the system settings.
